docker/dockerfiles/dispatcher

```diff
@@ -0,0 +1,26 @@
+FROM openjdk:8
+
+LABEL maintainer="wesley.monte@ccc.ufcg.edu.br"
+
+ENV MAIN_CLASS="org.fogbowcloud.saps.engine.core.dispatcher.SubmissionDispatcherMain"
+
+# Create bind mounts for these three entries
+ENV CONFIG_PATH="/dispatcher/dispatcher.conf"
+ENV LOG_PROPERTIES_PATH="/dispatcher/log4j.properties"
+ENV TEMP_STORAGE_DIR="/dispatcher/nfs"
+
+RUN apt-get update && apt-get install -y \
+    curl jq sed \
+    python \
+    python-swiftclient \
+    python-gdal \
+    python-shapely
+
+WORKDIR /dispatcher
+
+COPY ./target/saps-engine-0.0.1-SNAPSHOT.jar dispatcher.jar
+COPY ./target/lib  lib/
+COPY ./resources resources/
+COPY ./scripts  scripts/
+
+ENTRYPOINT java -Dlog4j.configuration=file:"${LOG_PROPERTIES_PATH}" -cp dispatcher.jar:lib/* "${MAIN_CLASS}" "${CONFIG_PATH}"
```

Comment:
just to confirm, what exactly the -cp part do?

Reply:
I thinks it means classpath

Reply:
Exact.

Reply:
Exact.